src/app/containers/Image/helpers/srcSetHelper.js

```diff
@@ -0,0 +1,27 @@
+const createSrcset = (originCode, locator, originalImageWidth) => {
+  const resolutions = [240, 320, 480, 624, 800];
+  const ichefUrl = 'https://ichef.bbci.co.uk/news';
```

Comment:
I feel like we should be able to reuse or refactor this in some way?

https://github.com/bbc/simorgh/blob/3f152e49d3f0d3da4a7fa28e6cfaf9d1427cbc04/src/app/containers/Image/index.jsx#L24-L29